src/span.cpp

```diff
@@ -148,8 +148,14 @@ void Span::FinishWithOptions(
   }
   tag = span_->meta.find(datadog_error_tag);
```

Comment:
There's a tag in opentracing ext for this, should we use that instead?

Reply:
Hi @cgilmour what is that TAG name? I can try for our testing.

I don't see other TAG relate to the error here: https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table